blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,228 @@
+
+
+variable "billing_account_id" {
+  description = "Billing account id."
+  type        = string
+}
+
+variable "bucket_name" {
+  description = "Create GCS Bucket."
+  type        = string
+  default     = null
+}
+
+variable "dataset_name" {
+  description = "Create BigQuery Datasets."
+  type        = string
+  default     = null
+}
+
+variable "env" {
+  description = "Environment (dev,stg,prd)."
+  type        = string
+  default     = "dev"
+}
+
+variable "folder_id" {
+  description = "Folder ID for the folder where the project will be created."
+  type        = string
+}
+
+variable "groups" {
+  description = "User groups."
+  type        = map(string)
+  default = {
+    data-scientists = "gcp-ml-ds"
+    ml-engineers    = "gcp-ml-eng"
+  }
+}
+
+variable "identity_pool_claims" {
+  description = "Claims to be used by Workload Identity Federation (i.e.: attribute.repository/ORGANIZATION/REPO)."
+  type        = string
+  default     = null
+}
+
+variable "kms_service_agents" {
+  description = "KMS IAM configuration in as service => [key]."
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "labels" {
+  description = "Labels to be assigned at project level."
+  type        = map(string)
+  default     = {}
+}
+
+
+variable "notebooks" {
+  description = "Vertex AI workbenchs to be deployed."
+  type = map(object({
+    owner                 = string
+    region                = string
+    subnet                = string
+    internal_ip_only      = bool
+    idle_shutdown_timeout = bool
+  }))
+  default = null
+}
+
+variable "org_policies" {
+  description = "Org-policy overrides at project level."
+  type = map(object({
+    inherit_from_parent = optional(bool) # for list policies only.
+    reset               = optional(bool)
+
+    # default (unconditional) values
+    allow = optional(object({
+      all    = optional(bool)
+      values = optional(list(string))
+    }))
+    deny = optional(object({
+      all    = optional(bool)
+      values = optional(list(string))
+    }))
+    enforce = optional(bool, true) # for boolean policies only.
+
+    # conditional values
+    rules = optional(list(object({
+      allow = optional(object({
+        all    = optional(bool)
+        values = optional(list(string))
+      }))
+      deny = optional(object({
+        all    = optional(bool)
+        values = optional(list(string))
+      }))
+      enforce = optional(bool, true) # for boolean policies only.
+      condition = object({
+        description = optional(string)
+        expression  = optional(string)
+        location    = optional(string)
+        title       = optional(string)
+      })
+    })), [])
+  }))
+  default  = {}
+  nullable = false
+}
+
+variable "organization_domain" {
+  description = "Organization domain."
+  type        = string
+}
+
+variable "prefix" {
+  description = "Prefix used for the project id."
+  type        = string
+  default     = null
+}
+
+variable "project_create" {
```

Comment:
can you switch to the approach we have on other blueprints?

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/4622b1a31f2738bfa150d25fe65d522aaab158dc/blueprints/data-solutions/gcs-to-bq-with-least-privileges/variables.tf#L45